### ISSUE TITLE
How to use this repo?

### ISSUE THREAD

**Issue by dipam7** [joonson/syncnet_python#31]
If I am correct, after downloading the models, I have to run

```
python run_pipeline.py --videofile /path/to/video.mp4 --reference name_of_video --data_dir /path/to/output
python run_syncnet.py --videofile /path/to/video.mp4 --reference name_of_video --data_dir /path/to/output
```

and the sync-corrected output will be stored in output_dir/pyavi/video.avi

Is this correct?

**Comment by joonson** [comment]
No, the output is not sync-corrected. It just gives you an offset and active speaker detection labels.

**Comment by dipam7** [comment]
How do I use this repo to sync correct my videos?

**Comment by dipam7** [comment]
Also the offsets.txt is not getting generated for me as mentioned in the README. Just 4 pckl files inside output_dir/pywork/reference_name/

**Comment by houdajun** [comment]
what is the --reference name_of_video? I only have a clip of video for sync, I am not sure what to specify for this input.

**Comment by houdajun** [comment]
> No, the output is not sync-corrected. It just gives you an offset and active speaker detection labels.

Do we need to sync it manually using the AV offset? How can we do it?

**Comment by Sushiman31** [comment]
> Also the offsets.txt is not getting generated for me as mentioned in the README. Just 4 pckl files inside output_dir/pywork/reference_name/

Hey , have you found the cause of the problem?
